docs/core/tools/dotnet-new.md

```diff
@@ -58,714 +28,688 @@ The command calls the [template engine](https://github.com/dotnet/templating) to
+| Solution File                                | `sln`                           |              | Solution                              | 1.0        |
 
 ## Options
 
-# [.NET Core 2.2](#tab/netcore22)
-
-`--dry-run`
-
-Displays a summary of what would happen if the given command were run if it would result in a template creation.
-
-`--force`
-
-Forces content to be generated even if it would change existing files. This is required when the output directory already contains a project.
-
-`-h|--help`
+- **`--dry-run`**
 
-Prints out help for the command. It can be invoked for the `dotnet new` command itself or for any template, such as `dotnet new mvc --help`.
+  Displays a summary of what would happen if the given command were run if it would result in a template creation. Available since .NET Core 2.2 SDK.
 
-`-i|--install <PATH|NUGET_ID>`
+- **`--force`**
 
-Installs a source or template pack from the `PATH` or `NUGET_ID` provided. If you want to install a prerelease version of a template package, you need to specify the version in the format of `<package-name>::<package-version>`. By default, `dotnet new` passes \* for the version, which represents the last stable package version. See an example at the [Examples](#examples) section.
+  Forces content to be generated even if it would change existing files. This is required when the output directory already contains a project.
 
-For information on creating custom templates, see [Custom templates for dotnet new](custom-templates.md).
+- **`-h|--help`**
 
-`-l|--list`
+  Prints out help for the command. It can be invoked for the `dotnet new` command itself or for any template, such as `dotnet new mvc --help`.
 
-Lists templates containing the specified name. If invoked for the `dotnet new` command, it lists the possible templates available for the given directory. For example if the directory already contains a project, it doesn't list all project templates.
+- **`-i|--install <PATH|NUGET_ID>`**
 
-`-lang|--language {C#|F#|VB}`
+  Installs a source or template pack from the `PATH` or `NUGET_ID` provided. If you want to install a prerelease version of a template package, you need to specify the version in the format of `<package-name>::<package-version>`. By default, `dotnet new` passes \* for the version, which represents the last stable package version. See an example at the [Examples](#examples) section.
+  
+  If a version of the template was already installed when you run this command, the template will be updated to the specified version, or to the latest stable version if no version was specified.
 
-The language of the template to create. The language accepted varies by the template (see defaults in the [arguments](#arguments) section). Not valid for some templates.
+  For information on creating custom templates, see [Custom templates for dotnet new](custom-templates.md).
```

Comment:
```suggestion
  For more information, see [Custom templates for dotnet new](custom-templates.md).
```

Reply:
It's ok to say what you'd get more information for.